test/common/buffer/buffer_fuzz.cc

```diff
@@ -165,6 +165,15 @@ class StringBuffer : public Buffer::Instance {
 
   std::string toString() const override { return std::string(data_.data() + start_, size_); }
 
+  void setWatermarks(uint32_t) override {
```

Comment:
followup: maybe worth adding this to the fuzzer when we merge classes.

Reply:
Added a TODO